test/integration/order.test.js

```diff
@@ -46,14 +46,15 @@ describe('order', function () {
 
       nock(`http://${IDENTITY_SERVICE_HOST}:${IDENTITY_SERVICE_PORT}`)
         .persist()
-        .get('/v1/members/supplier3000')
+        .get('/v1/self/5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty')
```

Comment:
thanks for updatiing, don't like using supplier3000 as it does not represent an uuid

Reply:
Hang on shouldn't this just be `/v1/self`?

Reply:
🤷‍♂️

Reply:
alright lets talk it through